### DISCUSSION TITLE
How can I use PyAEDT to control objects which is not created by PyAEDT in HFSS?

### DISCUSSION THREAD

**Discussion by Dicksky** [ansys/pyaedt#3175] [Q&A] [answered]
Hello. I would like to open an existing HFSS project and control the objects in the project, when I open the project using PyAEDT, I can not use the object's name to control it in PyAEDT, it tells me the name is undefined. How can I get control of objects using PyAEDT? 
Many thanks!

**Comment by fwlfHS** [comment, marked as answer]
It should be possible. Can you share a little bit more information on what you would like to do?
See https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.modeler.modeler3d.Modeler3D.get_object_from_name.html

**Reply by Dicksky** [reply]
Thank you very much!
